src/librustc_mir/transform/add_yk_swt_calls.rs

```diff
@@ -31,33 +51,24 @@ impl MirPass for AddYkSWTCalls {
             return;
         }
 
-        // Find the recorder function to call.
         let rec_fn_defid = tcx.get_lang_items(LOCAL_CRATE).yk_swt_rec_loc_wrap()
             .expect("couldn't find software trace recorder function");
 
-        // Types.
         let unit_ty = tcx.mk_unit();
         let u32_ty = tcx.types.u32;
         let u64_ty = tcx.types.u64;
 
-        // Each block is replaced by a new block whose terminator calls the recorder function.
-        let mut replace_blocks = Vec::new();
-
-        // The original blocks are copied and the recorder function returns to a copy.
+        let mut replacement_blocks = Vec::new();
```

Comment:
The convention in this function seems to be `blk` for `blocks`? I'm certainly a fan of the shorter version! It might be nice to find a shorter, and perhaps more accurate version of "replacement". Something like "shadow" perhaps?